acme.sh

```diff
@@ -1852,9 +1852,14 @@ _inithttp() {
       _ACME_CURL="$_ACME_CURL --cacert $CA_BUNDLE "
     fi
 
-    if _contains "$(curl --help 2>&1)" "--globoff"; then
+    if _contains "$(curl --help curl 2>&1)" "--globoff"; then
```

Comment:
Since someone version of curl, `curl --help <category>` needs a category argument.
`curl --help` never contains `--globoff` but `curl --help curl` do. It's also compatible with older version, the category argument will be ignored in older version.
Tested on:
- curl 7.85.0 (x86_64-apple-darwin22.0)
- curl 7.83.1 (x86_64-alpine-linux-musl)
- curl 7.85.0 (x86_64-openwrt-linux-gnu)
Backward compatible tested on:
- curl 7.61.1 (x86_64-redhat-linux-gnu)
- curl 7.29.0 (x86_64-redhat-linux-gnu)

Reply:
这个改动, 建议保留原先的判断,   如果原先没有结果的情况下, 再加上 category 再判断一次.  保持之前的兼容性.  有的系统上装的curl不知道是多老的版本.

Reply:
已经在curl 7.29.0（发布于2013年2月6日）验证过了兼容性，推测在同一个主版本curl 7.x上应该都不会有问题，而curl 7.1发布于2000年。（如果真有那么老的版本，可能比起help兼容性，更该担心的是TLS兼容性了）。
而且，一个正常的--help实现也应该是会把无关参数都无视的。
另外考虑到目前这个判断**已经**在大多数环境下失效了，包括neilpang/acme.sh的docker镜像。
说明即便判断错误也不会造成关键性的影响，因此我希望还是keep simple，不要做这个不必要的历史兼容判断。

Reply:
还是保持兼容性吧,  我见过各种奇奇怪怪的系统, 你永远也不知道对方是什么版本.  安全第一.